src/sentry/snuba/discover.py

```diff
@@ -961,12 +963,16 @@ def histogram_query(
     columns = [] if key_column is None else [key_column]
     limit = len(fields) * num_buckets
 
+    orderby = [histogram_alias]
+    if order_by:
+        orderby.insert(0, order_by)
+
     histogram_query = prepare_discover_query(
         selected_columns=columns + [histogram_column, "count()"],
         conditions=conditions,
         query=user_query,
         params=params,
-        orderby=[histogram_alias],
+        orderby=orderby,
```

Comment:
can we do something like this? orderby and order_by is far too similar
```suggestion
        orderby=(order_by if order_by else []) + [histogram_alias],
```